doc_src/cmds/math.rst

```diff
@@ -156,6 +156,8 @@ Functions
     the base-10 logarithm
 ``log2``
     the base-2 logarithm
+``logb(b,x)``
+    returns the logarithm with base b of x
```

Comment:
as mentioned, it's a bit odd that `logb()` means something else in libc but I suppose this short name fits in well.

Unlike C, we could add overloading
(i.e. support both `log(x)` and `log(base, x)` ).
I think that would be a viable alternative.
(The fact that `log(x)` is `log10` is weird.)

I think instead of

	$ math 'logb(5, 125)`
	3

you might be better off writing the unambiguous

	$ math 'log(125) / log(5)'
	3

but I guess this addition makes sense so I'm fine with it too